FirebaseCoreExtension.podspec

```diff
@@ -34,5 +34,5 @@ Pod::Spec.new do |s|
       "#{s.module_name}_Privacy" => 'FirebaseCore/Extension/Resources/PrivacyInfo.xcprivacy'
     }
 
-    s.dependency 'FirebaseCore', '~> 11.0'
+    s.dependency 'FirebaseCore', '>= 11.4.1', '< 12.0'
```

Comment:
Because FirebaseCoreExtension depends on FirebaseCore yet shares the same headers in its sources, they should be versioned together whenever these headers change.

Reply:
so how do i apply this now to my pod or code

Reply:
Hi @jenkinsernest, I added a comment for you over in https://github.com/firebase/flutterfire/issues/13527#issuecomment-2427832685

Reply:
ok thanks